workers/unity/Packages/com.improbable.gdk.core/GameObjectRepresentation/GameObjectDispatcherSystem.cs

```diff
@@ -0,0 +1,106 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using Unity.Entities;
+
+namespace Improbable.Gdk.Core
+{
+    /// <summary>
+    ///     Gathers incoming dispatcher ops and invokes callbacks on relevant GameObjects.
+    /// </summary>
+    [UpdateInGroup(typeof(SpatialOSReceiveGroup.GameObjectReceiveGroup))]
+    public class GameObjectDispatcherSystem : ComponentSystem
+    {
+        public readonly Dictionary<int, SpatialOSBehaviourManager> EntityIndexToSpatialOSBehaviourManager = new Dictionary<int, SpatialOSBehaviourManager>();
```

Comment:
Given that we iterate over the values of this dictionary every frame - is there a more efficient manner of storing this data?

Reply:
I will now also use a separate list to quickly iterate over the SpatialOSBehaviourManagers.